Detectors/ITSMFT/ITS/tracking/GPU/cuda/TrackingKernels.cu

```diff
@@ -68,9 +70,12 @@
 // #include "MatLayerCyl.cxx"
 
 // O2 track model
+#ifdef __HIPCC__
```

Comment:
I don't have the external stuff for HIP yet, so I had to leave your hack in to include the cxx files.
For CUDA, you get the TrackParCov and propagator from the linked external object library.